corehq/apps/app_manager/app_strings.py

```diff
@@ -184,11 +184,15 @@ def maybe_add_index(text):
                     lang, for_default=for_default, build_profile_id=build_profile_id)
                 search_again_label_audio = module.search_config.search_again_label.audio_app_string(
                     lang, for_default=for_default, build_profile_id=build_profile_id)
+                title_label = module.search_config.get_search_title_label(app, lang, for_default=for_default)
```

Comment:
Nitpick: move this into the `if` below. It's good to reduce variable [live time](http://www.rosscode.com/blog/index.php?title=variable_usage_span_and_live_time&more=1&c=1&tb=1&pb=1).